files/alertifyjs/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "alertifyjs",
+  "repo": "mohammadyounes/alertifyjs",
+  "files": {
+    "basePath": "build/",
+    "exclude": ["alertify.zip"]
```

Comment:
Please remove your exclude and use the following instead. (format it properly at first)
include: *.min.js, ./css/**/*.min.css